src/pages/Homepage/Homepage.tsx

```diff
@@ -107,6 +108,29 @@ export const Homepage = () => {
             ) : null}
           </div>
         </div>
+        <div className={styles.projectsCard}>
+          {projectsData.map(
+            (
+              {
+                projectTitle,
+                projectDescription,
+                projectPictureUrl,
+                projectUrl,
+                projectInfoUrl,
+              },
+              index,
+            ) => (
+              <ProjectsCard
+                projectTitle={projectTitle}
+                projectDescription={projectDescription}
+                projectPictureUrl={projectPictureUrl}
+                projectUrl={projectUrl}
+                projectInfoUrl={projectInfoUrl}
+                cardOrientation={index % 2 === 0 ? "normal" : "reversed"}
+              />
+            ),
+          )}
```

Comment:
You can do something like this to avoid writing each of the props.

```ts
{projectsData.map((props, index) => (
    <ProjectsCard
      {...props}
      cardOrientation={index % 2 === 0 ? "normal" : "reversed"}
    />
  ),
)}
```

Also rename `cardOrientation` to `cardDirection`